src/tests/index.test.js

```diff
@@ -93,7 +93,7 @@ async function validateClient(evolv, options, uid, sid) {
   expect(await evolv.getConfig('web')).to.be.an('undefined');
   expect(await evolv.getConfig('web.ab8numq2j')).to.be.an('undefined');
   expect(await evolv.getConfig('web.ab8numq2j.am94yhwo2')).to.be.an('undefined');
-  expect(contextChangedSpy).to.have.been.called(3);
+  expect(contextChangedSpy).to.have.been.called(8);
```

Comment:
I'm curious this changed. Were our tests not passing previously?

Reply:
The updates i made, cause more calls to be made

Reply:
Now that we are updating the context object with the events and active keys. This 1) causes more context changed events to fire and 2) causes more events to be sent back to the analytics endpoint. This is by design as we want these things to occur when events or keys are added.